functions/HttpTriggerAPIUsersId/__init__.py

```diff
@@ -0,0 +1,53 @@
+import logging
+import pypyodbc
+import os
+import azure.functions as func
+import json
+
+
+def main(req: func.HttpRequest) -> func.HttpResponse:
+    db_server = os.environ["ENV_DATABASE_SERVER"]
+    db_name = os.environ["ENV_DATABASE_NAME"]
+    db_username = os.environ["ENV_DATABASE_USERNAME"]
+    db_password = os.environ["ENV_DATABASE_PASSWORD"]
+    driver = '{ODBC Driver 17 for SQL Server}'
+    connectionString = "Driver={};Server={};Database={};Uid={};Pwd={};Encrypt=yes;TrustServerCertificate=no;Connection Timeout=30;".format(
+        driver, db_server, db_name, db_username, db_password)
+
+    id = req.params.get('id')

```

Comment:
What is this id for? Whose id is it?

Reply:
@lbrown51 this id is the id that gets passed as a parameter when this function gets called.
I'm also wondering if we need to support creating a user in one of these API functions.

Reply:
this is the userId. 
Natalia suggested renaming this "id" to "userId" for clarification. The way I have it right now makes it confusing. Will update it

Reply:
Awesome! I think that makes a lot of sense. Currently, it could be misinterpreted as a separate id of some kind.

Reply:
Yes! good practice to keep names for the parameters same across the board. I used userId and taskId in my code